pkg/providers/anthropic/chat_stream.go

```diff
@@ -2,15 +2,102 @@ package anthropic
 
 import (
 	"context"
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
 
 	"glide/pkg/api/schemas"
 	"glide/pkg/providers/clients"
 )
 
+type Client struct {
+	APIKey string
+}
+
 func (c *Client) SupportChatStream() bool {
-	return false
+	return true
+}
+
+func (c *Client) ChatStream(ctx context.Context, chatReq *schemas.ChatRequest) (clients.ChatStream, error) {
+	apiURL := "https://api.anthropic.com/v1/complete"
+	requestBody := map[string]interface{}{
```

Comment:
We have this interesting method to combines configs provided by user in Glide provider YAML config + incoming request data to map them into a real Anthropic request:
https://github.com/EinStack/glide/blob/develop/pkg/providers/anthropic/chat.go#L82-L88

Would be great to leverage it here.